agir/notifications/urls.py

```diff
@@ -10,6 +11,7 @@
 router = DefaultRouter()
 router.register(r"webpush", WebPushDeviceAuthorizedViewSet)
 router.register(r"apple", APNSDeviceAuthorizedViewSet)
+router.register(r"apple", GCMDeviceAuthorizedViewSet)
```

Comment:
C'est `android`, non ?
```suggestion
router.register(r"android", GCMDeviceAuthorizedViewSet)
```